### ISSUE TITLE
Word documents are marked as Spam by Gmail

### ISSUE THREAD

**Issue by UlyC** [thinkst/canarytokens#151]
<img width="640" alt="image" src="https://user-images.githubusercontent.com/36838240/190295687-8d94e172-ac6e-487c-832e-3c7f9c96f0f7.png">

**Comment by jayjb** [comment]
Hi @UlyC,

Thank you for reporting this. It seems gmail isn't happy about embedding links that are automatically resolved when the document is opened. We will keep an eye on this and see if we figure a nice way around it.

**Comment by UlyC** [comment]
> It seems gmail isn't happy about embedding links that are automatically resolved when the document is opened.

I tried building my own service, using a different domain, and it passed Gmail's checks.

Another important point is that if the link in the attachment is HTTP, it will definitely be marked as Spam.

And currently, the default link used by Canarytokens is HTTP, I tried changing it to HTTPS but it still gets flagged as spam, it looks like it's because your domain is flagged by them.
